File: pytools/data/data.code.php.client/ReqSceneEnter.php
<?php
namespace protocol;

use network\Packet;


class ReqSceneEnter
{
	private $doorId = 0;


	public function encode()
	{
		$packet = new Packet();
		$packet->writeU32($this->doorId);
		return $packet->encode(Msg::$P_REQ_SCENE_ENTER);
	}


	public function setDoorId($doorId)
	{
		$this->doorId = $doorId;
	}

}

File: pytools/data/data.code.java.client/ReqSceneEnter.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqSceneEnter
{
	private int door_id;


	public byte[] encode()
	{
		Packet packet = new Packet();
		packet.writeInt(door_id);
		return packet.encode((short)Msg.P_REQ_SCENE_ENTER);
	}



	public void setDoorId(int door_id)
	{
		this.door_id = door_id;
	}

}
